data/json/mapgen/shelter.json

```diff
@@ -388,6 +505,125 @@
       "palettes": [ { "distribution": [ [ "shelter_used", 10 ], [ "shelter_vandal", 10 ] ] } ]
     }
   },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "weight": 70,
+    "//": "Copy of shelter_2_vandal, unfinished before the cataclysm. No roof.",
+    "om_terrain": [ "shelter_unfinished_2" ],
+    "object": {
+      "fill_ter": "t_floor",
+      "rows": [
+        "                        ",
+        "                        ",
+        "                        ",
+        "                 ~~~~   ",
+        "                 ~~~~   ",
+        "                        ",
+        "                        ",
+        "   ||--              |  ",
+        "   |*|..                ",
+        "   |....                ",
+        "  |--|..                ",
+        "  |......               ",
+        "  :.......              ",
+        "  |.............        ",
+        "  |.......|--|.......|  ",
+        "  |........>>........|  ",
+        "  |........>>........|  ",
+        "  :.......|--|.......:  ",
+        "  |..................|  ",
+        "  |..................|  ",
+        "  |--..............--|  ",
+        "   |...............c|4  ",
+        "   |--------|......c|   ",
+        "            |-:-+-:-|   "
+      ],
+      "palettes": [ "shelter_vandal" ],
+      "terrain": { "~": "t_dirt" },
+      "items": { "~": { "item": "lumberyard", "chance": 90, "repeat": [ 3, 5 ] } }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "weight": 50,
+    "//": "Infested spawn for Compromized Shelter scenario - copy of shelter_1. Window on right wall near basement door removed for scenario balance, sorry. Spawns are a bit sporadic, but usually give at least 1 running enemy and ~4-7 others at 0.6 density.",
+    "om_terrain": [ "shelter_infested" ],
+    "object": {
+      "fill_ter": "t_floor",
+      "rows": [
+        "    !!!!!!!!!!!!!!!!    ",
+        "    `!!!!`!!!!`!!!!`    ",
+        "    `!!!!`!!!!`!!!!`    ",
+        "    `!!!!`!!!!`!!!!`    ",
+        "    `!!!!`!!!!`!!!!`    ",
+        "    `!!!!`!!!!`!!!!`    ",
+        "    `!!!!`!!!!`!!!!`    ",
+        "    &&&|---++---|&&&    ",
+        "       |........|       ",
+        "    |-:|..7.....|       ",
+        "    |...........|:-|    ",
+        "    |.bbbbb.....c.6|    ",
+        "    |...........c..|    ",
+        "    :.bbbbb.....c..:    ",
+        "    |...........cc.|    ",
+        "    |.bbbbb........|    ",
+        "    |.......l|||...|    ",
+        "    |.-----.l|*=...|    ",
+        "    :.......l|||..8|    ",
+        "    |.b.b.b...c|-+-|    ",
+        "    |.b7b.b...c|..l|    ",
+        "    |.b.b.b...c|>.l:    ",
```

Comment:
Just noticed that the window here at 577 should be moved to line 573 to match shelter_1!